pkgs/development/compilers/fasmg/default.nix

```diff
@@ -32,7 +32,7 @@ stdenv.mkDerivation rec {
         bin = "source/macos/fasmg";
         asm = "source/macos/fasmg.asm";
       };
-    }.${system} or (throw "Unsopported system: ${system}");
+    }.${system} or (throw "Unsupported system: ${system}");
```

Comment:
Sorry, meant this! :see_no_evil:

Reply:
Would be overkill in my opinion to pull it out into another PR, could move it to a separate commit but I personally think it's just a typo so it's fine as is